### DISCUSSION TITLE
Pie Chart breakdown of current power and consumed energy by device

### DISCUSSION THREAD

**Discussion by Parker-McDonnell** [tnmendes/watt#133] [Ideas]
Just an idea. It would be great if you could switch from the existing tile view of each outlet (with small text that indicate power consumption) to a pie chart view that would show you what percentage of the current total power is being consumed by which device.

**Comment by tnmendes** [comment]
Hi there,

Could you explain in more detail how you envision the pie chart view replacing the existing tile view? Any additional context or visual representation you can provide would greatly assist me in grasping the concept.
